File: Cap 04/Topico 4.5/Topico 4.5.1/Decisao Sequencial/decseq.swift
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.5.1
 * 
 * Programa ..: OPERACAO COM DECISAO SEQUENCIAL
 */

import Foundation

var N: Int = 0

print("DECISAO SEQUENCIAL\n\n")
print("Entre valor inteiro <N>: ", terminator: "")
if let inputN = readLine() {
    N = Int(inputN) ?? 0
}
if N == 1 {
    print("voce entrou o valor 1")
}
if N == 2 {
    print("voce entrou o valor 2")
}
if N < 1 {
    print("voce entrou valor muito baixo")
}
if N > 2 {
    print("voce entrou valor muito alto")
}

File: Cap 04/Topico 4.5/Topico 4.5.1/Decisao Sequencial/decseq.jl
#=
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.5.1
 *
 * Programa ..: OPERACAO COM DECISAO SEQUENCIAL
=#

println("DECISAO SEQUENCIAL")
println("")

print("Entre valor inteiro <N>: ")
N = parse(Int, readline())

if (N == 1)
  println("voce entrou o valor 1")
end

if (N == 2)
  println("voce entrou o valor 2")
end

if (N < 1)
  println("voce entrou valor muito baixo")
end

if (N > 2)
  println("voce entrou valor muito alto")
end

